### ISSUE TITLE
Why the normal alignment block was partially cut out

### ISSUE THREAD

**Issue by jxlabWzZ** [EichlerLab/pav#32]
Hi, i successfully obtained SV results by using pav (to compare two T2T assemblies with default parameters). 
However, when I checked the pre-cut or final alignment file, I found that the alignment block after the cut-step was partially disappeared. I don't know why this is so and whether it will affect the final result.

For example, the following case show missing ~6.7Mb(from 13328139 to 14018782):
(1) align/pre-cut/aligned_tig_h1.bed.gz
![截屏2023-04-10 下午10 55 29](https://user-images.githubusercontent.com/54127791/230927531-a1b22286-94ea-4f3c-88e2-f7a6e3a0a6d6.png)

(2) align/aligned_tig_h1.bed.gz
![截屏2023-04-10 下午10 56 00](https://user-images.githubusercontent.com/54127791/230927532-f8e525fa-09ed-441b-adc9-186ddd6eb548.png)

**Comment by paudano** [comment]
PAV does not allow redundant alignments in either reference or contig space. If more than one contig is aligned to that region of the reference, it trims one of them back removing as many small variants as possible. For example, if one contig aligned to the region is full of indels and SNPs and the other is not, the one with indels and SNVs is trimmed back and the cleaner alignment represents that region of the reference. The same thing happens to redundantly mapped contigs (i.e. the same contig locus aligned to two separate places of the genome). If contigs are equally well mapped, then trimming favors trimming leftward in reference space to right-align SVs that might be causing the alignment to break into multiple records.

PAV does this for two reasons, to find the true SV breakpoint where there's imperfect homology (often causes redundant alignments affecting breakpoints) and to remove false variant calls from improperly placed alignments.

I don't see anything in this alignment that looks like it should have been trimmed this heavily, although it could be another alignment record not in these four. If you see QUERY_TIG_POS or QUERY_TIG_END overlapping another record in the pre-cut BED, then that's what caused it. If not, can you share the BED files, or is this just hg38 against hs1 (I can try to reproduce this if so)?

**Comment by jxlabWzZ** [comment]
Thank you for your reply.
It`s non-human genomes and is not publicly available at present.
we extract chr1 info as follows:
[precut.chr1.txt](https://github.com/EichlerLab/pav/files/11195095/precut.chr1.txt)
[final.chr1.txt](https://github.com/EichlerLab/pav/files/11195098/final.chr1.txt)

**Comment by jxlabWzZ** [comment]
Correction: 670kb region is missing, not 6.7Mb.

**Comment by jxlabWzZ** [comment]
Correction: ~670kb region is missing, not 6.7Mb.

**Comment by paudano** [comment]
I'll take a look at this and see what I can find. What reference are you using?

**Comment by paudano** [comment]
Let me know if I can help with this any further. If you find evidence of a bug where PAV trimmed away alignments it should have, please share what you can.

**Comment by jxlabWzZ** [comment]
I'm sorry, I forgot to reply you in time. 
My reference genome and query genome have not been uploaded to NCBI and are not publicly available at present.
Fortunately, I found the cause of this bug, that is, both my single reference chromosome (the biggest one) and single query chromosome have ~300Mb in length, which exceeds 2 ^ 31-1 (2147483647), causing an error in the current version of numpy. Therefore, after I split the single query chromosome into two contig (<200Mb per contig), it was solved perfectly.
